gitfiti.py

```diff
@@ -207,7 +207,7 @@ def load_images(img_names):
         return loaded_imgs
 
 
-def get_calendar(username, base_url):
+def retrieve_contributions_calendar(username, base_url):
     """retrieves the GitHub commit calendar data for a username"""
     base_url = base_url + 'users/' + username
```

Comment:
Also, if the second argument is about allowing to use this with other providers besides GitHub, the URL path should probably be configurable as well (even if, say, GitLab, might provide data under the same path pattern).